### ISSUE TITLE
Hanging block type: lanterns, chandeliers, etc.

### ISSUE THREAD

**Issue by WolfieWaffle** [EnviroMine/EnviroMine-1.7#132]
I am making a mod pack with both enviromine and thermal expansion. The problem is, certain blocks like lumium lamps can hang from the bottom of a block. These blocks support blocks above them and can make cheap easy infinite bridges using this exploit. If you turn physics on for the lamps, they break when hanging from the ceiling.

I have tried the config options, yes. None of them seem to accomplish what I'm after. This probably isn't a huge issue, but it would be helpful to avoid cheating the physics system.

Anyways, how I think it would work is this:

- Hanging blocks are stable if there is a stable block above them
- If the above block is unstable (e.g. floating) and physics are triggered, this block would break, allowing the block above it to fall.
- Hanging blocks do not add any support for blocks, and are not counted in the connected block count
- Hanging blocks would have an option to hang from walls and sit on the floor.

I hope you consider my suggestion, and I hope I explained it clearly enough!

**Comment by GenDeathrow** [comment]
Its actually not a bad suggestion very well thought out. Because @funwayguy it currently tearing down for a 1.8 build I can't promise when something like this would be implemented . But ill take a look at it here and there. I have an idea of how to do it code wise. Kinda like how falling blocks work with torches. If I figure that out it should be easy.

**Comment by WolfieWaffle** [comment]
Thanks! I always love to see compatibility and stuff. By 1.8 update do you mean minecraft 1.8? And if so, would there be a possibility of backwards comparability or a back port, since most modpacks are usually about a version behind?

**Comment by GenDeathrow** [comment]
Yes minecraft 1.8. And 1.8 will have a complete rewrite. And will be split up into separate mods so it wont be backward comparable. But we will prolly still do work with 1.7.10 for a while till it loses interest. But that hasn't really been talked about yet. So nothing is set in stone.

**Comment by WolfieWaffle** [comment]
Ok, so it is likely that the next new features will have a 1.7.10 version?
I was just thinking for modpacks and such. If your not too busy, what are
the new mods going to be split into?

**Comment by Grim-Ascensions** [comment]
I'm a fan of this idea as well. Nice suggestion Wolfie!
